3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -53,61 +58,71 @@ enrich {
       # 'nsq' for reading / writing to a Nsq topic
       # 'stdin' for reading from stdin and writing to stdout and stderr
       enabled =  {{sinkType}}
+      enabled =  ${?ENRICH_STREAMS_SOURCE_SINK_ENABLED}
 
       # Region where the streams are located (AWS region, pertinent to kinesis sink/source type)
       # region = {{region}}
+      # region = ${?ENRICH_STREAMS_SOURCE_SINK_REGION}
 
       ## Optional endpoint url configuration to override aws kinesis endpoints,
       ## this can be used to specify local endpoints when using localstack
       # customEndpoint = {{kinesisEndpoint}}
+      # customEndpoint = ${?ENRICH_STREAMS_SOURCE_SINK_CUSTOM_ENDPOINT}
 
       # AWS credentials
       # If both are set to 'default', use the default AWS credentials provider chain.
       # If both are set to 'iam', use AWS IAM Roles to provision credentials.
       # If both are set to 'env', use env variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
       # aws {
-      #   accessKey = iam
-      #   secretKey = iam
+      #   accessKey = "iam"
+      #   accessKey = ${?ENRICH_STREAMS_SOURCE_SINK_AWS_ACCESS_KEY}
+      #   secretKey = "iam"
+      #   secretKey = ${?ENRICH_STREAMS_SOURCE_SINK_AWS_SECRET_KEY}
       # }
 
       # Maximum number of records to get from Kinesis per call to GetRecords
       # maxRecords = 10000
+      # maxRecords = ${?ENRICH_MAX_RECORDS}
 
       # LATEST: most recent data.
       # TRIM_HORIZON: oldest available data.
       # "AT_TIMESTAMP": Start from the record at or after the specified timestamp
       # Note: This only effects the first run of this application on a stream.
       # (pertinent to kinesis source type)
       # initialPosition = TRIM_HORIZON
+      # initialPosition = ${?ENRICH_STREAMS_SOURCE_SINK_INITIAL_POSITION}
 
       # Need to be specified when initial-position is "AT_TIMESTAMP".
       # Timestamp format need to be in "yyyy-MM-ddTHH:mm:ssZ".
       # Ex: "2017-05-17T10:00:00Z"
       # Note: Time need to specified in UTC.
-      initialTimestamp = "{{initialTimestamp}}"
+      initialTimestamp = {{initialTimestamp}}
```

Comment:
this had quotes originally, that's why the tests are failing :+1: 

If you're interested:

- templating goes on here: https://github.com/snowplow/snowplow/blob/master/3-enrich/stream-enrich/integration-tests/src/test/scala/com.snowplowanalytics.snowplow.enrich.stream/PiiEmitSpec.scala#L77
- the example config is turned into a resource here: https://github.com/snowplow/snowplow/blob/master/3-enrich/stream-enrich/project/BuildSettings.scala#L56-L58

Reply:
it's true for other places too, some quotes have been unnecessarily remove